modules/bim/game/main/include/bim/game/arena.hpp

```diff
@@ -39,7 +36,7 @@ namespace bim::game
     bool is_solid(std::uint8_t x, std::uint8_t y) const;
     void set_solid(std::uint8_t x, std::uint8_t y);
 
-    std::span<const bim::game::static_wall> static_walls() const;
+    std::vector<bim::game::static_wall> static_walls() const;
```

Comment:
This one should not have changed, but it's completely my fault! I should not have listed `m_static_walls` in the issue.

The point of this function is to have a direct access to the list of static walls such that I can display them all at once in `bim::axmol::app::online_game`. If we switch to a table we end up with a worse design:
 - There's a dynamic allocation to build the vector at every iteration,
 - There's an indirection to get the static walls via the optional.
 - They are not grouped in memory and must be copied from the table to the array.

My apologies for having listed it in the issues. This function and `m_static_walls` should be restored in their previous state.